packages/lex_service/src/lexicon/resolvers/lexicon.resolver.ts

```diff
@@ -16,6 +18,18 @@ export class LexiconResolver {
     return this.lexiconService.create(lexicon);
   }
 
+  @Mutation(() => Lexicon)
+  @UseGuards(AuthGuard)
+  async lexiconUpdate(
+    @Args('lexicon', { type: () => String }, LexiconPipe) lexicon: Lexicon,  
```

Comment:
Very close, but we will want to make a new type for the update, something like `LexiconUpdate` which has each field as optional. I'm happy to meet to talk about this more if you like too!